pkg/trace/stats/statsraw.go

```diff
@@ -52,8 +89,15 @@ type RawBucket struct {
 	keyBuf strings.Builder
 }
 
+// PayloadKey is the key of a stats payload.
```

Comment:
This isn't really documentation 😛

Reply:
😆 true, actually, I don't need to export that anymore.